### PR TITLE
feat(notifications): make notifications NPM packages public

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: plugins/notifications-backend/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
-  "private": true,
+  "private": false,
```
Comment (invincibleJai) [inline plugins/notifications-backend/package.json:7]: we can just remove it over specifying it as false

File: plugins/notifications/package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@backstage/plugin-notifications-frontend",
+  "name": "@backstage/plugin-notifications",
   "description": "Frontend part of the notifications providing the Notifications API, Notifications page, left-side menu item and other REACT components.",
   "version": "0.1.0",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
-  "private": true,
+  "private": false,
```
Comment (invincibleJai) [inline plugins/notifications/package.json:8]: same as above, we can avoid this over making it false

Comment (mareklibra) [reply plugins/notifications/package.json]: done

**Review by invincibleJai — APPROVED** [review]
/lgtm

**Pull request merged**